README.md

```diff
@@ -1,7 +1,37 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
You can remove the `*generated with [DocToc](https://github.com/thlorenz/doctoc)*` ad and it knows not to put it back next time doctoc is run.